ArjunaJTA/recovery/pom.xml

```diff
@@ -27,35 +27,34 @@
     <version>6.0.0.Final-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <description>Recovery example
-	</description>
+  <description>Recovery example</description>
   <modelVersion>4.0.0</modelVersion>
-  <artifactId>recovery</artifactId>
+  <artifactId>jta_recovery</artifactId>
   <packaging>jar</packaging>
   <name>Recovery Examples</name>
   <dependencies>
-    <!-- Embedded Hornetq -->
+    <!-- Embedded Artemis ActiveMQ -->
     <dependency>
-      <groupId>org.hornetq</groupId>
-      <artifactId>hornetq-jms-server</artifactId>
-      <version>2.4.9.Final</version>
-    </dependency>
+      <groupId>org.apache.activemq</groupId>
+      <artifactId>artemis-jakarta-server</artifactId>
+      <version>2.25.0</version>
```

Comment:
The artifact `org.jboss.narayana.jta:jms` in the main repo is using 2.24.0 (ArjunaJTA/jms/pom.xml), does this quickstart rely on that module and if so I would choose the same version?

Is this dependency unused because of the comment 
```
 #Jakarta TODO, integrate back JMS recovery example
```
in the file ArjunaJTA/recovery/run.sh

Reply:
Yes, it was unused at the moment. Now I uncommented the JMS recovery example so it is used. I then agree to set the version to 2.24.0. Thanks

Reply:
Great, so the PR looks good.